utility/utils.py

```diff
@@ -533,7 +533,7 @@ def custom_ceph_config(suite_config, custom_config, custom_config_file):
     return full_custom_config
 
 
-def email_results(results_list, run_id, trigger_user, run_dir, suite_run_time, send_to_cephci=False):
+def email_results(results_list, run_id, trigger_user, run_dir, suite_run_time, info, send_to_cephci=False):
```

Comment:
*args, **kwargs, addressed same in the previous comment in calling module.

Reply:
IMO, the argument could be a dictionary having the required and/or optional keys. It would address Sunil's concern of having it flexible. This would require proper docstring though...